test/conftest.py

```diff
@@ -56,10 +46,16 @@ def get(
         return response
 
     def assert_404(self, url: str):
-        raise NotImplementedError()
+        response = self._get_response(url, lambda f: f())
+        assert response.status == 404
+        content = decode_response(response)
+        assert isinstance(content, dict) and isinstance(content["message"], str)
```

Comment:
Now checks if the response returns `{message: ...}` for all backends. I moved the implementation in `BaseServer` as it is the same for flask and fastapi. 

It could be the same for tornado as well if there was not a `HTTPClientError` to catch... More below.